### PR TITLE
[feat] added product page and listings

### PR DESCRIPTION
**Pull request by saurabhan**
### Product Listing Page
image

### Product Info Page
image

### CODE REVIEW

**Review by DivyanshSareen — APPROVED** [review]
Code is clean and readable. Great work on the UI. Good to merge

File: frontend/src/components/CategoryListing/CategoryListing.jsx [hunk 1]
```diff
@@ -14,11 +14,11 @@ const CategoryListing = () => {
             </div>
 
             <div className="mt-6 grid grid-cols-1 gap-y-6 sm:grid-cols-2 sm:grid-rows-2 sm:gap-x-6 lg:gap-8">
-              <div className="group aspect-w-2 aspect-h-1 rounded-lg overflow-hidden sm:aspect-h-1 sm:aspect-w-1 sm:row-span-2">
```
Comment (Megha-Joshi) [inline frontend/src/components/CategoryListing/CategoryListing.jsx:17]: Nicely used utility classes.

**Review by Megha-Joshi — APPROVED** [review]
Naming convention is impressive. Loved the UI. Good to merge!!

File: frontend/src/pages/ProductInfo.jsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
+import { useState } from "react";
+import {
+  CheckIcon,
+  StarIcon,
+} from "@heroicons/react/solid";
+import { products } from "../db/productslist";
+
+const reviews = { average: 4, totalCount: 1624 };
+
+
```
Comment (zoyatanweer) [inline frontend/src/pages/ProductInfo.jsx:10]: remove extra spacing

**Review by zoyatanweer — APPROVED** [review]
You have written the code very nicely. Great work on the UI. Good to commit this. I like how you have used utility classes. Good to merge.

**Review by sumitk-singh — APPROVED** [review]

**Comment by sumitk-singh** [comment]
Appreciation: 
Class name used in a meaningful way, 
Clean code, easy to read and understand
Overall UI look good to me.

**Pull request merged**